pom.xml

```diff
@@ -321,6 +332,226 @@
         <artifactId>requirejs</artifactId>
         <version>${requirejs.version}</version>
       </dependency>
+
+      <dependency>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-beans</artifactId>
+        <version>${spring-version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-context</artifactId>
+        <version>${spring-version}</version>
+        <exclusions>
+          <exclusion>
+            <artifactId>*</artifactId>
```

Comment:
Why the exclusion at this level?

Reply:
Eh, probably made sense somewhere in the process or maybe was some side effect of a copy&paste ....looking at it right now, it doesn't make sense at all. Will review all identified occurrences.